.github/workflows/build-test.yml

```diff
@@ -13,11 +13,7 @@ name: Build and Test
 # Clippy is special because we're using actions-rs/clippy-check which is able to surface clippy failures on
 # PR bodies
 
-on:
-  push:
-    branches: [ main ]
-  pull_request:
-    branches: '*'
+on: [push]
```

Comment:
You'll need to remember to undo this change before merging.

(optional: I usually forget to undo such changes for testing purposes later on. So I started to keep a "real" branch that is ready for merging, and a "testing" branch that rebases 1 or 2 commits on top of the "real" branch ([example](https://github.com/echeran/icu4x/pull/18)). Whatever is easier for you)